draft-normington-jsonpath.md

```diff
@@ -240,13 +252,12 @@ The JSONPath tool in question should:
 * be lightweight in code size and memory consumption.
 * be runtime efficient.
 
-## Overview of JSONPath Expressions
+## Overview of JSONPath Expressions {#overview}
```

Comment:
The anchor `#overview` clashes with the default anchor for the _other_ `Overview` section.